### PR TITLE
Its not a wonder women [blog post]

### PR DESCRIPTION
**Pull request by nader77**
it_s_not_a_wonder__women_

gizra_s_blog

### CODE REVIEW

**Comment by nader77** [comment]
@OritGeron @RachelBaram Deployed:
http://www.gizra.com/content/its-not-a-wonder-women/

/cc @amitaibu @IshaDakota @ordavidil

File: src/_posts/2017-03-08-its-not-a-wonder-women.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+{% include thumbnail.html image_path="/assets/images/posts/its-not-a-wonder-women/wonder-woman-and-the-guys.jpg" caption="One of the guys." %}
+
+When I started working at Gizra, there was only one other female employee in the office. I was kind of relieved - I was in a familiar environment. A year ago, Amitai challenged this with a facebook post (it’s in Hebrew) that ended-up going viral. In the job posting, he talked about wanting more diversity - especially women - in our company. His words made me think. Feel. He nudged me out of my comfort zone. It left me curious (and nervous) to find out how this will change our office environment.
```
Comment (amitaibu) [inline src/_posts/2017-03-08-its-not-a-wonder-women.md]: btw, why not link the facebook post to @RachelBaram post? http://www.gizra.com/content/job-post-goes-viral/

Comment (OritGeron) [inline src/_posts/2017-03-08-its-not-a-wonder-women.md]: Sure! @nader77

Comment (amitaibu) [inline src/_posts/2017-03-08-its-not-a-wonder-women.md]: I'll create a PR

Comment (amitaibu) [inline src/_posts/2017-03-08-its-not-a-wonder-women.md]: https://github.com/Gizra/gizra.com/pull/586

**Pull request merged**